docs/extras/nandrestore.md

```diff
@@ -1,36 +1,39 @@
-## Restore a nand backup to the switch
+## Restore a NAND backup to the switch
 
-!!! danger "Warning:"	
-	-This will reset all of your saves, games, system version and other system settings back to a point when you made the the nand backup. Keep this in mind, as you probably don't have to restore a nand backup unless you have bricked your switch or want to go back online safely after using cfw.
+!!! Danger "Warning:" 	
```

Comment:
Inconsistent, this should be `danger`, not `Danger`.

Reply:
all have been changed to lowercase